### PR TITLE
Fixed EOL to unix - some where windows.  Updated Local developer stac…

### PR DESCRIPTION
**Pull request by alexanderward**
…k.  Fixed minor bug where filenames weren't being updated

### CODE REVIEW

**Review by gregelin — CHANGES_REQUESTED** [review]

File: deployment/local/ssh/ssh_host_dsa_key [hunk 1]
```diff
@@ -0,0 +1,21 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```
[the remark this answers is not in the document]
Comment (alexanderward) [reply deployment/local/ssh/ssh_host_dsa_key:1]: It's for local development purposes only.   It is needed for the reverse SSH connection for external IDE interpreters.  It is not used in any other stack or for any other purpose.

[the remark this answers is not in the document]
Comment (alexanderward) [reply deployment/local/ssh/ssh_host_dsa_key:1]: I'll look into another option.

[the remark this answers is not in the document]
Comment (alexanderward) [reply deployment/local/ssh/ssh_host_dsa_key]: Resolved with a generation technique.

File: deployment/local/Dockerfile [hunk 2]
```diff
@@ -35,9 +43,6 @@ WORKDIR /usr/src/app
 RUN python3 -m pip install --upgrade pip
 RUN pip3 install ipdb
 
-# install wkhtmltopdf for generating PDFs, thumbnails
```
[the remark this answers is not in the document]
Comment (alexanderward) [reply deployment/local/Dockerfile:38]: Moved back out to it's own spot.

**Comment by gregelin** [comment]
Excellent!

Greg Elin
CEO, GovReady PBC
p: 917-304-3488
e: ***@***.***

Sent via Superhuman iOS ( ***@***.*** )

**Comment by alexanderward** [comment]
https://github.com/GovReady/govready-q/issues/1437

File: deployment/local/remote_interpreter/python_env.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+source /usr/src/app/deployment//local/remote_interpreter/env_var.sh
```
Comment (gregelin) [inline deployment/local/remote_interpreter/python_env.sh:2]: Should there be a double slash in this path?

Comment (alexanderward) [reply deployment/local/remote_interpreter/python_env.sh]: Nope.  It works, but it's a typo.  Removed.

**Review by gregelin — APPROVED** [review]

**Review by gregelin — APPROVED** [review]
looks good

**Pull request merged**